en/physics/physics-event.md

```diff
@@ -33,11 +33,11 @@ There are three types of Trigger Events:
 
 Where the collision pairs that can generate trigger events are:
 
-| Type | Static rigid body | Kinematic rigid body | Dynamic rigid body |
+| Type | **Static rigid body** | **Kinematic rigid body** | **Dynamic rigid body** |
```

Comment:
I'm sorry. I was not clear. You don't need to bold the table headings, just the key of each row.

Change this back to: 

| Type | Static rigid body | Kinematic rigid body | Dynamic rigid body |